### ISSUE TITLE
EXIF GPS data not read (piwigo-openstreetmap)

### ISSUE THREAD

**Issue by keefian** [Piwigo/Piwigo#520]
Hi,

This issue was first opened on the piwigo-openstreetmap github page : 
[https://github.com/xbgmsharp/piwigo-openstreetmap/issues/109#](url)

It was confirmed by **xbgmsharp** unfortunately no solution was found.

Some of my photos EXIF GPS data are not read on my Piwigo installation (**Ubuntu 16.04 server with latest PWG package**).

I have attached 2 photos as an example : 
[photo.zip](https://github.com/Piwigo/Piwigo/files/441210/photo.zip)

For both the photos : 
- the EXIF GPS data were set using Geosetter software
- the EXIF GPS data are correctly retrieved by various software (like Picasa or FastStone Image Viewer)

But on my PWG server, only EXIF GPS data of photo2.jpg are retrieved correctly :-(

**Comment by wittypluck** [comment]
Hello,

I had the same issue and debugged it to a problem in parsing some images which have makernotes in exif.
See : http://piwigo.org/forum/viewtopic.php?pid=164428.

I found a bug in PHP bug tracker which seems to be the cause of the issue :
https://bugs.php.net/bug.php?id=72682

I was using php 7.0.8. An upgrade to php 7.0.10 fixed the problem for me.

For ubuntu 16.04 I used this PPA :
https://launchpad.net/~ondrej/+archive/ubuntu/php

Hope it helps !
